db/migrations/0001_initial.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.0.2 on 2023-06-06 13:30
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Actor',
```

Comment:
change ' to ", Use Black

Reply:
0001_initial.py is autogenerate file, should I edit it?
Also what do you mean by saying "Use Black"? Can you give some more information.